.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
       - name: Lint Ruby files and RBI shims
         run: bundle exec rubocop
-  build:
+  build_ruby:
```

Comment:
Not a blocker, but we might want to look at skipping the Ruby tests if only the `vscode` part is touched, and vice versa.

Reply:
I think we don't want to do that. Part of the advantage of having both in the same repository is that we'll be able to have real integration tests that boot make requests from the extension to the server booted from the same repo.

Running all tests will help us catch changes that might break the integration.